Controller/PostController.php

```diff
@@ -23,9 +23,11 @@
 class PostController extends Controller
 {
     /**
+     * @param Request $request
+     *
      * @return RedirectResponse
      */
-    public function homeAction()
+    public function homeAction(Request $request = null)
```

Comment:
Should work with `Request $request = null`.

But, right, this could be indeed be simplier.